docs.go

```diff
@@ -19,10 +18,11 @@ func splitDocs(docs string) (title, desc string) {
 	return
 }
 
-// rapiDocTemplate is the template used to generate the RapiDoc.  It needs two args to render:
-// 1. the title
-// 2. the path to the openapi.yaml file
-var rapiDocTemplate = `<!doctype html>
+// RapiDocHandler renders documentation using RapiDoc.
+func RapiDocHandler(router *Router) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "text/html")
+		w.Write([]byte(fmt.Sprintf(`<!doctype html>
```

Comment:
Can you give an example of how this should look? I'm having trouble understanding this comment :frowning_face:

Reply:
Ah, sorry. There are blocks of HTML Scattered through the tests that look mostly similiar, and I thought it would improve readability to pull those HTML snippets into a fixture file of some kind. Does that make more sense?

Reply:
I think each of these blocks is unique and comes from different documentation project's READMEs/demos. I'll take a look around for anything that might be duplicated but for the most part keeping these separate makes them easier to update (just a copy/paste from upstream if/when needed).